networking/external_dns_operator/nw-installing-external-dns-operator.adoc

```diff
@@ -0,0 +1,33 @@
+[id="nw-installing-external-dns-operator_{context}"]
+= Installing the External DNS Operator
+
+You can install External DNS Operator using the Red Hat OpenShift Container Platform Operator Hub.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+* You have created the operand `external-dns` namespace.
+* You have created the `rolebinding` and `clusterrolebinding`.
```

Comment:
FYI...we can see "Prerequisites and Requirements" below on console as well, so maybe we can add the related commands here. 
```
The ExternalDNS Operator has minimal cluster level permissions. Unfortunately OLM doesn't allow the local role creation in the operand namespace yet. Therefore make sure the following commands are run against your cluster before the installation of the operator:

Create the operand namespace: oc create ns external-dns
Grant the operator access to manage operand resources: oc apply -f https://raw.githubusercontent.com/openshift/external-dns-operator/main/config/rbac/extra-roles.yaml
```

The two commands are:
```
$ oc create ns external-dns
$ oc apply -f https://raw.githubusercontent.com/openshift/external-dns-operator/main/config/rbac/extra-roles.yaml
```
```

Reply:
Addressed